app/models/match_prioritization/days_homeless_populations.rb

```diff
@@ -20,6 +20,15 @@ def self.prioritization_for_clients(scope, match_route:) # rubocop:disable Lint/
       scope.order(family, vets, is_youth, age_youth, days)
     end
 
+    def self.supporting_column_names
+      [
+        :family_member,
+        :veteran,
+        :is_currently_youth,
```

Comment:
this should use the new method right?
```suggestion
        :calculated_is_currently_youth,
```